src/app/components/content/home/home.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="text-center">
     <p class="font-weight-bolder h3 mb-5 mx-4" id="intro-text">Last gehad van (seksuele) intimidatie tijdens het uitgaan?</p>
     <button
-      class="font-weight-bolder btn btn-lg btn-light rounded-pill border border-dark shadow mt-5"
```

Comment:
any particular reason why you removed the rounded corners and font-weight styling?

Reply:
I styled the button the same way I styled it in the partner component for consistency. The rounded corners are added with the "btn" class.

Reply:
> The rounded corners are added with the "btn" class.
I know that, 

I'm just wondering why you adjusted the styling of a button that was already conform styling that Rosanne already had provided. Also, checking the designs for the partner page also shows some deviations between the designs provided by Rosanne and our implementation.